deployment/ccip/changeset/v1_6/init_chain_for_testing.go

```diff
@@ -0,0 +1,66 @@
+package v1_6
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+)
+
+var _ deployment.ChangeSet[InitChainForTestingConfig] = InitChainForTesting
+
+// InitChainForTestingConfig is a configuration struct for InitChainForTesting.
+type InitChainForTestingConfig struct {
+	ChainSelector uint64
+}
+
+// InitChainForTesting sequences all changesets required to add a chain to CCIP from scratch.
+// This particular changeset activates the new lanes on test routers.
+func InitChainForTesting(e deployment.Environment, config InitChainForTestingConfig) (deployment.ChangesetOutput, error) {
+	newAddresses := deployment.NewMemoryAddressBook()
+
+	output, err := changeset.DeployPrerequisitesChangeset(e, changeset.DeployPrerequisiteConfig{
+		Configs: []changeset.DeployPrerequisiteConfigPerChain{
+			changeset.DeployPrerequisiteConfigPerChain{
+				ChainSelector: config.ChainSelector,
+			},
+		},
+	})
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to run DeployPrerequisitesChangeset: %w", err)
+	}
+	newAddresses.Merge(output.AddressBook)
+
+	/*
+		Throughout the course of this changeset, aggregate all MCMS proposals into a single MCMS proposal.
+		Output this MCMS proposal and all new addresses at the end of execution.
+
+		1. Run changesets against new chain WITHOUT MCMS
+		- DeployPrerequisitesChangeset
+		- DeployChainContractsChangeset
+		- SetOCR3OffRampChangeset
+		- UpdateFeeQuoterPricesChangeset
+		- UpdateFeeQuoterDestsChangeset
+		- UpdateOffRampSourcesChangeset (TestRouter = true)
+		- UpdateOnRampsDestsChangeset (TestRouter = true)
+		- UpdateRouterRampsChangeset (for both inbound and outbound traffic, TestRouter = true)
```

Comment:
There is also deployMCMs contracts, saveExistingTokenAddresses, saveFeedAddreses